substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -1020,7 +1020,9 @@ def _native_image_launcher_extra_jvm_args():
             destination="bin/<exe:native-image>",
             jar_distributions=["substratevm:SVM_DRIVER"],
             main_class=_native_image_launcher_main_class(),
-            build_args=driver_build_args,
+            build_args=driver_build_args + [
+                '-H:IncludeResources=com/oracle/svm/driver/launcher/.*',
```

Comment:
Could we use a `resource-config.json` for this?

Reply:
It's not. But to get to that point we would need to move **all** native-image config to `META-INF/native-image` not just this `resource-config.json` and I'm not sure if this should be part of this PR here (which is adding container/launcher support for bundles and not driver cleanup). I'm fine with a followup PR that does this cleanup.